packages/js/src/integrations-page/site-kit-integration.js

```diff
@@ -153,12 +216,17 @@ export const SiteKitIntegration = ( {
 				isActive={ successfullyConnected }
 			>
 				<span className="yst-flex yst-flex-col yst-flex-1">
-					{ successfullyConnected ? <>
-						<SuccessfullyConnected />
-						<Button className="yst-w-full" id="site-kit-integration__button" variant="secondary" onClick={ toggleDisconnectModal }>
-							{ __( "Disconnect", "wordpress-seo" ) }
-						</Button>
-					</> : <Button className="yst-w-full" id="site-kit-integration__button" { ...buttonProps[ currentStep ] } /> }
+					{ successfullyConnected && capabilities.viewSearchConsoleData && <SuccessfullyConnected /> }
+
+					<NoPermissionWarning
+						capabilities={ capabilities }
+						currentStep={ currentStep }
+					/>
+					<Button
+						className="yst-w-full"
+						id="site-kit-integration__button"
+						{ ...buttonProps[ successfullyConnected ? 4 : currentStep ] }
```

Comment:
4?

Reply:
I added the props to the array

Reply:
I meant this is a random magical number.

That you now made a name for even.